src/main/java/us/spaceclouds42/playtime_tracker/mixin/ServerPlayNetworkHandlerMixin_TimeTracker.java

```diff
@@ -60,7 +60,7 @@ private void trackTime(CallbackInfo ci) {
             )
     )
     private void updateLastActionTime(PlayerMoveC2SPacket packet, CallbackInfo ci) {
-        if (packet instanceof PlayerMoveC2SPacket.LookOnly) {
+        if (packet instanceof PlayerMoveC2SPacket.LookAndOnGround) {
```

Comment:
Is this a yarn change, or a Minecraft change?

Reply:
https://github.com/FabricMC/yarn/pull/2104/files#diff-d2ce97667e1545218aa9533d59f51d6d82d7fbc240474ec42a8b5b593d5b5571L39-R49

Reply:
Then this entire thing is just a yarn remapping, no? :thinking: